portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,17 +36,34 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private static final int REQUESTED_COMMENTS_LIMIT = 100;
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Sort comments by date from newest to oldest
+    int maxCommentsNumber;
+
+    // Try to convert the value from the 'max-comments' field of the query string to int
+    try {
+      maxCommentsNumber = Integer.parseInt(request.getParameter("max-comments"));
```

Comment:
Can we check whether a parameter exist before pasing?

Reply:
Done!